### ISSUE TITLE
AttributeError: 'CountVectorizer' object has no attribute 'get_feature_names'

### ISSUE THREAD

**Issue by jmorenobl** [ddangelov/Top2Vec#307]
Hello,

Following the examples in the readme I created this code:

```
documents = df["cleaned_message"].tolist()
model = Top2Vec(
    documents,
    embedding_model="universal-sentence-encoder",
    speed="learn",
    workers=multiprocessing.cpu_count() - 1,
)

print(f"Num topics: {model.get_num_topics()}")
```
And that is throwing the following error:

AttributeError: 'CountVectorizer' object has no attribute 'get_feature_names'

**Comment by flydsc** [comment]
It seems that the new sklearn api had removed 'get_feature_names', they put a new one called 'get_feature_names_out'.

ref: https://scikit-learn.org/stable/modules/generated/sklearn.feature_extraction.text.CountVectorizer.html#sklearn.feature_extraction.text.CountVectorizer

**Comment by levrone1987** [comment]
I had the same problem, but it persists even after

` pip install top2vec -U`

What should I do? @flydsc @jmorenobl

**Comment by flydsc** [comment]
> 

It seems that they are working on that issues and gonna release a new version.

so for a quick fix: you may just modify your local code in the package;

for long term, plz wait for the official update

**Comment by levrone1987** [comment]
Thanks, @flydsc 
I did this, but now I have the following error (regardless of which text):

` raise ValueError(f"A min_count of {min_count} results in "
ValueError: A min_count of 50 results in all words being ignored, choose a lower value.`

The code:

`
# for german language, use a pretrained model
top2vec_model = Top2Vec(corpus, speed="learn", workers=8, embedding_model='distiluse-base-multilingual-cased')

count_id = 0
similar_top2vec = top2vec_model.search_documents_by_documents(doc_ids=[count_id])
`

**Comment by ddangelov** [comment]
Fixed in version `1.0.28`
